### ISSUE TITLE
Torrents aren't using public IP networks, as expected.

### ISSUE THREAD

**Issue by JayKayOb** [BiglySoftware/BiglyBT#2125]
Both Public IP network and I2P network are checked under Options|Connection|Networks. but under Networks from the right-click drop down menu, only I2P network is selected when a torrent starts downloading.
I'm not certain but believe the problem started with an update, as all torrents used to use both public IP and I2P networks.
Also, the default auto-stop setting is Stop Immediately, when I believe it should be defaulting to seeding until a share ratio of 2.5 is reached.
I usually use Magnet links by default.
How do I change the defaults for both settings?

**Comment by parg** [comment]
Options->Connection->Networks are the default settings - these can be overridden

For example, the Magnet URI Handler plugin has its own defaults (Options->Plugins->Magnet URI Handler)

Also a magnet can include explicit networks via an &net=<network> argument.

**Comment by JayKayOb** [comment]
Thanks for your reply.
The Magnet URI handler also has Public and I2P networks checked.
There's no &net= argument in this particular torrent.
In a very strange development, having checked the I2P network box in the torrent's drop-down menu, I discovered it unchecked again some time later. This happened three times before I eventually managed to complete the download, taking several hours to download a file which should have taken less than an hour.
